api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -225,6 +227,12 @@ message GrpcJsonTranscoder {
   // :ref:`proto descriptor set <config_grpc_json_generate_proto_descriptor_set>`.
   bool convert_grpc_status = 9;
 
+  // Specifies a proto transformer extension that transforms grpc response status into a custom protobuf format.
+  // This field is effective only when ``convert_grpc_status`` is set to true.
+  // The target custom prototype type must be in ``proto_descriptor``.
+  // This field should be an extension of ``envoy.proto_transformer``.
```

Comment:
Should be of the form:
`[#extension: envoy.abc.def.ghi]`

Reply:
I noticed there is presubmit checking extension_metadata.yaml about the extension existence. For `envoy.common.proto_transformer`, we only have the extension API no real extension now. Is skipping that presubmit the only way?